src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs

```diff
@@ -164,6 +164,7 @@ public void WaitForPeerProcess()
         }
 
         [Fact]
+        [ActiveIssue(15844)]
```

Comment:
This has only failed on Unix, right? We should only disable it there then, e.g. adding TestPlatforms.AnyUnix